src/BizHawk.Client.EmuHawk/MainForm.Events.cs

```diff
@@ -615,15 +615,25 @@ private void ViewSubMenu_DropDownOpened(object sender, EventArgs e)
 		private void WindowSizeSubMenu_DropDownOpened(object sender, EventArgs e)
 		{
 			var windowScale = Config.GetWindowScaleFor(Emulator.SystemId);
-			foreach (ToolStripMenuItem item in WindowSizeSubMenu.DropDownItems)
+			foreach (ToolStripMenuItem item in WindowSizeSubMenu.DropDownItems.OfType<ToolStripMenuItem>())

```

Comment:
Did you mean
```suggestion
			foreach (var item in WindowSizeSubMenu.DropDownItems.Cast<ToolStripMenuItem>())
```
?
You shouldn't have to change it though.

Reply:
No, it's filtering out the separator

Reply:
I think the type check on `Tag` is enough for that, but you could also do something like:
```csharp
foreach (var item in WindowSizeSubMenu.DropDownItems.Cast<ToolStripItem>())
{
	if (item is not ToolStripSeparator && item.Tag is int itemScale)
	{
		// ...
	}
}
```
The confusing type hierarchy means it's not clear what `.OfType<ToolStripMenuItem>()` does just from looking at it; normally it'd be fine.

Reply:
The items need to be cast to `ToolStripMenuItem` to access the `Checked` property so there needs to be some kind of additional cast. I can change it to `item is ToolStripMenuItem menuItem` or something inside the loop if that's clearer

Reply:
Yes, do that